package.json

```diff
@@ -6,14 +6,17 @@
   "author": "Nikhil Arora <narora200@gmail.com>, Saumya Tiwari <saumyatiwari.29@gmail.com>",
   "license": "MIT",
   "scripts": {
-    "start": "echo \"starting the-remotants backend\" && nodemon src"
+    "start": "echo \"starting the-remotants backend\" && nodemon src",
```

Comment:
Don't have a `start` pointing to a development tool. 
Use `yarn start:dev` for running any development specific task since `yarn start` may be used by other services by default.